src/main/java/org/folio/processing/mapping/defaultmapper/processor/functions/NormalizationFunction.java

```diff
@@ -206,6 +209,9 @@ private String concatSubFields(JsonObject ruleParameter, List<Subfield> subfield
     @Override
     public String apply(RuleExecutionContext context) {
       String subFieldValue = context.getSubFieldValue();
+      if (subFieldValue == null || subFieldValue.isEmpty()) {
```

Comment:
propose to check if `subfieldValue` length is less than 6 + 1 instead of checking for empty string in order to cover also the case when subfield value length is less than 7 (which is the case that we faced).
And for the CHAR_SELECT function above propose checking if subFieldValue length is less than TO_PARAMETER + 1